plugins/parodos/src/stores/slices/notificationsSlice.ts

```diff
@@ -76,9 +111,12 @@ export const createNotificationsSlice: StateCreator<
       set(state => {
         const notification = state.notifications.get(id);
         if (newState === 'READ' && notification) {
-          state.notifications = state.notifications.set(id, {
-            ...notification,
-            read: true,
+          unstable_batchedUpdates(() => {
+            state.notifications = state.notifications.set(id, {
+              ...notification,
+              read: true,
+            });
+            state.notificationsLoading = false;
```

Comment:
If the condition is false, the app will stuck with `notificationsLoading == true`

Reply:
great spot!